packages/webapp/components/layouts/FooterNavBarLayout.tsx

```diff
@@ -27,7 +27,7 @@ export default function FooterNavBarLayout({
   return (
     <>
       {children}
-      {showNav && <div className={post ? 'h-40' : 'h-28'} />}
+      {showNav && <div className={post ? 'h-40' : 'h-16'} />}
```

Comment:
Think `h-28` was for old mobile nav, which caused double scrollable content on mobile.